cluster/saltbase/salt/kube-dns/skydns-svc.yaml.sed

```diff
@@ -12,8 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM BASEIMAGE
-MAINTAINER Tim Hockin <thockin@google.com>
-ADD kube2sky /
-ADD kube2sky.go /
-ENTRYPOINT ["/kube2sky"]
+# This file should be kept in sync with cluster/addons/dns/skydns-rc.yaml.in
```

Comment:
skydns is a different container https://hub.docker.com/r/skynetservices/skydns/, I think we should just call this kubedns-rc/svc etc.